packages/amplify-graphql-api-construct-tests/src/__tests__/sql-models-common.ts

```diff
@@ -19,77 +21,696 @@ export const testGraphQLAPI = async (options: {
   const outputs = await cdkDeploy(projRoot, '--all', { postDeployWaitMs: ONE_MINUTE });
   const { awsAppsyncApiEndpoint: apiEndpoint, awsAppsyncApiKey: apiKey } = outputs[name];
 
-  const description = 'todo description';
+  const appSyncClient = new AWSAppSyncClient({
+    url: apiEndpoint,
+    region,
+    disableOffline: true,
+    auth: {
+      type: AUTH_TYPE.API_KEY,
+      apiKey,
+    },
+  });
 
-  const result = await graphql(
-    apiEndpoint,
-    apiKey,
-    /* GraphQL */ `
-      mutation CREATE_TODO {
-        createTodo(input: { description: "${description}" }) {
+  // Test: check CRUDL on contact table with default primary key - postgres
+  const createTodo = async (description: string, id?: string): Promise<Record<string, any>> => {
```

Comment:
tl;dr: Refactor this method to make it more readable.

1. Here and throughout: Rather than inlining these helpers in the body of the test function, please extract them into separate methods. It will make it a lot more understandable to have it broken out instead of one long function.
2. Similarly: Rather than sequentially performing a bunch of semi-related tests in the same method, consider breaking each individual test into its own method so it's easier to understand what's happening in the body of the `testGraphQLAPI` method. Ideally, a reader would be able to quickly read through the `testGraphQLAPI` method from top to bottom, understand what cases are being covered, and navigate to the detailed test methods if they wanted to explore further. As written, all that detail is inline, making it harder to understand.

Reply:
Refactor `describe` and `test` blocks with native `beforeAll` and `AfterAll` for structuring functionalities. Implement a generic CRUDL class to remove duplicate/similar helpers.